<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtcDialog</class>
 <widget class="QDialog" name="RtcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>412</width>
     <height>304</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Date DD/MM/YY</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="dayButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 255);
border: 0px</string>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="monthButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 255);
border: 0px</string>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="yearButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 255);
border: 0px</string>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hour HH:MM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="hourButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 255);
border: 0px</string>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="minuteButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 170, 255);
border: 0px</string>
            </property>
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="UpButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="stretcher_res.qrc">
            <normaloff>:/imag/resources/arrow_up.png</normaloff>:/imag/resources/arrow_up.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DwnButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="stretcher_res.qrc">
            <normaloff>:/imag/resources/arrow_down.png</normaloff>:/imag/resources/arrow_down.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="doneButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Done!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="stretcher_res.qrc"/>
 </resources>
 <connections/>
</ui>
